app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -33,26 +35,36 @@ class FileSetsController < ApplicationController
     # GET /concern/file_sets/:id
     def edit
       initialize_edit_form
+    rescue WorkflowAuthorizationException
```

Comment:
this this redundant with line 3?

Reply:
It is. I had problems with this at first and thought it was a bug with 'rescue_from' (which I researched), but it isn't. Removed.